src/common/Logging/Log.cpp

```diff
@@ -222,6 +222,11 @@ void Log::outMessage(std::string const& filter, LogLevel level, std::string&& me
     write(std::make_unique<LogMessage>(level, filter, std::move(message)));
 }
 
+void Log::_outMessageFmt(std::string const& filter, LogLevel level, std::string&& message)
+{
+    write(std::make_unique<LogMessage>(level, filter, std::move(message)));
```

Comment:
Isn't the std::move here redundant? There's no move constructor in the LogMessage and the message parameter is already a rvalue so we're casting rvalue to rvalue